example/index.tsx

```diff
@@ -22,16 +20,6 @@ type TabParamList = {
   fifth: undefined;
 };
 
-const Stack: TypedNavigator<StackParamList, typeof StackNavigator> = {
```

Comment:
I think we still want annotation for our types to work:

```ts
const MyStack: TypedNavigator<StackParamList, typeof StackNavigator> = Stack;
```

Reply:
yeah, but they doesn't stop working?